radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -26,7 +26,8 @@ static int antennaInitialized = 0;
 /** Track Number of times the antenna attempted to deploy*/
 static int antennaDeploymentAttempts = 0;
```

Comment:
I know you didn't write the code originally, but you should fix the 2 variables (`RAntennaI2Caddress`, `RAntennaStatus`) above:
- Add the `static` keyword so they're private and not public globals
- Remove the `R` prefix, and change to camelCase e.g. `antennaI2cAddress` and `antennaStatus`
- Add single space after comma in struct definition of `antennaI2cAddress` on line 18